src/main/java/Kvothe.java

```diff
@@ -0,0 +1,204 @@
+import java.util.Scanner;
+
+public class Kvothe {
+
+    private static final int MAXTASKS = 100;
+    private static Task[] tasks = new Task[MAXTASKS];
+    private static int tasksIndex = 0;
+
+    public static void main(String[] args) {
+        String greeting = "Hello! I'm Kvothe.\n"
+                + "\t\tWhat can I do for you?\n";
+
+        String bye = "Bye. Hope to see you again soon!";
+
+        echo(greeting);
+
+        interact();
+
+        echo(bye);
```

Comment:
There are 2 different levels of abstraction here. You can consider making a new method for the string declaration.